js-exercises/cache-function/cacheFunction.js

```diff
@@ -0,0 +1,15 @@
+const cacheFunction = (fn) => {
+  let cache = {}
+  return (...args) => {
+    let n = args[0]
+    if (n in cache) {
+      return cache[n]
+    } else {
+      let result = fn(n)
```

Comment:
You can use the spread operator to spread all the arguments. 
```suggestion
      let result = fn(...args)
```